packages/react-native-bridge/android/react-native-bridge/build.gradle

```diff
@@ -0,0 +1,180 @@
+buildscript {
+    ext.kotlinVersion = '1.3.61'
+
+    repositories {
+        jcenter()
+        google()
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
+    }
+}
+
+plugins {
+    id "com.jfrog.bintray"
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'kotlin-android'
+apply plugin: 'maven-publish' // this enables publishing via Bintray
+
+// import the `readReactNativeVersion()` function
+apply from: 'https://gist.githubusercontent.com/hypest/742448b9588b3a0aa580a5e80ae95bdf/raw/8eb62d40ee7a5104d2fcaeff21ce6f29bd93b054/readReactNativeVersion.gradle'
+
+// import the `readHashedVersion()` function
+apply from: 'https://gist.githubusercontent.com/hypest/ceaf20a8e7d9b8404e4a5ff2e6c36650/raw/e1460a128e4b9863963410d719c7d44c3adefd02/readHashedVersion.gradle'
+
+// import the `waitJitpack()` function
+apply from: 'https://gist.githubusercontent.com/hypest/f526fe0775dedce0ce0133f1400d22a4/raw/0008b271a0d28fc79957fd3c2a027f57e98f796a/wait-jitpack.gradle'
+
+group='com.github.wordpress-mobile.gutenberg-mobile'
```

Comment:
Similar to the plugin version, I wonder if there's a way to share this value across `-aztec` and `-bridge`?

The answer is probably no, since they're separate projects from separate packages, I doubt there's a way to do it, and that's okay. I'm mainly nitpicking / thinking out loud. 👍

Reply:
We might be able to do this when/if we try to pull out common logic to a new gradle file, but unfortunately I couldn't get to it as part of this project.